Commit for code review

nikolayhsu opened the pull request:
Project pushed for code review.

nikolayhsu committed:
Commit for code review

nikolayhsu committed:
Update production bundle file name
Replace index checkbox with radio
Retain categories, indexes, and filters after search
Remove Reset button
Scroll to result table after search

gvanimurthy commented on package.json:
```diff
@@ -0,0 +1,58 @@
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://nikolay_hsu@bitbucket.org/nikolay_hsu/dsiti-search-function.git"
```
change this to current repo

gvanimurthy commented on package.json:
```diff
@@ -0,0 +1,58 @@
+  "bugs": {
+    "url": ""
+  },
+  "homepage": "https://bitbucket.org/nikolay_hsu/dsiti-search-function",
```
change this to current home page

nikolayhsu committed:
Update download links

nikolayhsu committed:
Update repo and homepage url

gvanimurthy commented on src/public/index.html:
```diff
@@ -0,0 +1,110 @@
+</div>
+<!--#include virtual="/assets/includes/global/footer.html"-->
+<!-- endnoindex -->
+<script type="text/javascript" src="//cdn.datatables.net/1.10.13/js/jquery.dataTables.min.js"></script>
```
@raymedia are we allowed to use CDN?

gvanimurthy commented on src/public/index.html:
```diff
@@ -0,0 +1,110 @@
+<!--#include virtual="/assets/includes/global/footer.html"-->
+<!-- endnoindex -->
+<script type="text/javascript" src="//cdn.datatables.net/1.10.13/js/jquery.dataTables.min.js"></script>
+<link rel="stylesheet" href="//cdn.datatables.net/1.10.13/css/jquery.dataTables.min.css" />
```
@raymedia are we allowed to use CDN?

nikolayhsu committed:
Fix table rendering bug
Update project name

ghost commented:
How we are testing the application?  I could not find any spec for Unit test and  E2E test.

nikolayhsu commented:
I am working on unit test at the moment.

GangaDeviC committed:
added a new folder fo orderonline form

nikolayhsu committed:
Remove leave feedback link
Update message in the table when no results found
Add Clear Research button

Add itemTitleField in category data
Modify datatable to allow x scroll
Update text

GangaDeviC committed:
modified the order_online form

nikolayhsu committed:
Scroll to Search by when selecting an index
Fix the issue when an index is not unchecked after changing category

GangaDeviC committed:
Style and text changes

GangaDeviC committed:
made changes to order online form

nikolayhsu committed:
Remove data/categories.js and replace with public/categories.json
Modify category.service.js to fetch source json on load
Update error messages
Cosmetic changes

GangaDeviC committed:
made changes to qsa-req-form-code-with-js.html

GangaDeviC committed:
made link changes to order online form

nikolayhsu committed:
Add unit test cases
Modify testing environment
Bug fix
